libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -750,18 +749,18 @@ void AC_AutoTune::get_poshold_attitude(float &roll_cd_out, float &pitch_cd_out,
       position. This ensures that autotune doesn't have to deal with
       more than 2.5 degrees of attitude on the axis it is tuning
      */
-    float target_yaw_cd = degrees(atan2f(pdiff.y, pdiff.x)) * 100;
+    float target_yaw_rad = atan2f(pos_error_neu_cm.y, pos_error_neu_cm.x);
     if (axis == AxisType::PITCH) {
         // for roll and yaw tuning we point along the wind, for pitch
         // we point across the wind
-        target_yaw_cd += 9000;
+        target_yaw_rad += cd_to_rad(9000);
```

Comment:
we could use degrees(90) instead of cd_to_rad(9000) if we want to reduce the usage of centi-degrees in constants.  just an idea though